<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWindow</class>
 <widget class="QDialog" name="SignupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ĐĂNG KÝ TÀI KHOẢN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailEdit">
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="placeholderText">
       <string>Họ và tên</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordEdit">
      <property name="placeholderText">
       <string>Mật khẩu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="confirmPasswordEdit">
      <property name="placeholderText">
       <string>Nhập lại mật khẩu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signupButton">
      <property name="text">
       <string>Đăng ký</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Quay lại đăng nhập</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
